src/main/content/_assets/js/guide-static.js

```diff
@@ -76,14 +76,26 @@ $(document).ready(function () {
                 .replace(".html", "")
                 .replace("/guides/", "");
             var host = window.location.hostname;
-            var skills_network_url =
-                host === "openliberty.io"
-                    ? data.skillNetworkUrl
-                    : data.stagingSkillNetworkUrl;
-            skills_network_url = skills_network_url.replace(
-                "{projectid}",
-                guide_name
-            );
+            var skills_network_url;
+
+            if (data.courses && data.courses.guide_name) {
+                skills_network_url =
+                    host === "openliberty.io"
+                        ? data.skillNetworkDomain
+                        : data.stagingSkillNetworkDomain;
+                skills_network_url += "/" + data.courses.guide_name;
+            } else {
+                // Guide is not in the list of courses
```

Comment:
Could you add a comment that this is the old deprecated format, and it's just here until all guides are moved to the new format?